sdk/purview/azure-analytics-purview-sharing/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
+
+### 1.0.0-beta.1 (2023-02-15-preview)
```

Comment:
The date of release should be in the brackets so people know when we released the version of the library

Reply:
Thanks.  Addressed this in the latest push.